rootfs/templates/wrapper

```diff
@@ -139,7 +139,10 @@ function use() {
 		--name "${DOCKER_NAME}"
 		--rm
 		--env KUBERNETES_API_PORT=${GEODESIC_PORT})
-	set -o pipefail
+	# With set -o pipefail, the pipe below fails ~50% of the time on Mac due to a bug in bash.
+	# With set +o pipefail, this bug is mitigated, and the only difference is the behavior when `docker ps` fails.
+	# If `docker ps` fails, then it is likely that any further docker action will fail too, so it does not matter.
+	set +o pipefail
```

Comment:
why `set -o pipefail` changed to `set +o pipefail`?

Reply:
`	# With set -o pipefail, the pipe below fails ~50% of the time on Mac due to a bug in bash.`